examples/evalboards/TMC5130/TMC5130_MicroStep.py

```diff
@@ -134,20 +138,35 @@
             measured = measured + [(STEP, CUR_A, CUR_B)]
             motor.move_to(1, 1000)
             time.sleep(0.1)
+            print('\rProgress: ', round(i/1025*100, 2), '%', end='')              #shows the progress
```

Comment:
please follow the files print-style and do this
```py
print("\rProgress: {0}%".format(round(i/1025*100, 2)), end="")
```
BTW: instead for the rounding with round you could specify the number of digits printed
```py
print("\rProgress: {0:.2f}%".format(i/1025*100), end="")
```
The `.2f` tells the print function that you expect a float at this position and want to round to 2 digits after the comma.